### PR TITLE
Add remote control command next track and previous track for iOS

### PR DESCRIPTION
**Pull request by Pikaurd**
Fast forward and rewind buttons can be displayed on lock screen while playing audio.
Added button to show previous and next songs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## [next]
 
+## audioplayers 0.14.3
+- Add next and previous command for ios
```
Comment (luanpotter) [inline CHANGELOG.md]: please add under next, don't create a new version yet

File: pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: audioplayers
 description: A flutter plugin to play multiple audio files simultaneously
-version: 0.14.2
+version: 0.14.3
```
Comment (luanpotter) [inline pubspec.yaml]: remove this change, we will release later

File: example/lib/player_widget.dart [hunk 3]
```diff
@@ -153,14 +155,17 @@ class _PlayerWidgetState extends State<PlayerWidget> {
...
-            imageUrl: 'url or blank',
-            forwardSkipInterval: const Duration(seconds: 30), // default is 30s
-            backwardSkipInterval: const Duration(seconds: 30), // default is 30s
-            duration: duration,
-            elapsedTime: Duration(seconds: 0));
+          title: 'App Name',
+          artist: 'Artist or blank',
+          albumTitle: 'Name or blank',
+          imageUrl: 'url or blank',
+          // forwardSkipInterval: const Duration(seconds: 30), // default is 30s
```
Comment (luanpotter) [inline example/lib/player_widget.dart]: why did you comment this out?

File: CHANGELOG.md (hunk 1 above)
Comment (Pikaurd) [reply CHANGELOG.md:6]: I'll change it and commit again.

File: example/lib/player_widget.dart (hunk 3 above)
Comment (Pikaurd) [reply example/lib/player_widget.dart:162]: That is just for testing, I'll add some comment.

In iOS, we cannot display both fast forward and rewind and the previous and next.

**Comment by sirily11** [comment]
I think this is a very good implementation which I tested on my iPhone and running like a charm.  I hope this can be merged as soon as possible.

**Comment by luanpotter** [comment]
@Pikaurd are you working on this? otherwise I will address the comments and merge.

**Comment by muarachmann** [comment]
@Pikaurd this is cool, tested and it works good. Can't wait for this to be merged :)

**Comment by Pikaurd** [comment]
> @Pikaurd are you working on this? otherwise I will address the comments and merge.

Recently busy I'll finish it in few days.

**Comment by Pikaurd** [comment]
Sorry guys, too busy to finish this PR. I will submit again when ready .

**Pull request closed without merging**